### DISCUSSION TITLE
Formily 设计器 git地址？求解答

### DISCUSSION THREAD

**Discussion by CoderDMBai** [alibaba/formily#2803] [Q&A 2.x]
请问Formily 设计器的git地址是什么？
预览页面：https://designable-antd.formilyjs.org/
之前的地址是：https://github.com/alibaba/formily/tree/formily_next/designable/antd 现在失效了。。。

**Comment by tangming2010** [comment]
https://github.com/alibaba/designable

**Reply by CoderDMBai** [reply]
您好！您这个地址我也下载过 在master分支启动后是下图页面，这个也不是Formily设计器的页面呀。。求解答
![image](https://user-images.githubusercontent.com/30764148/151641685-7031fe60-7dff-4373-9487-d7ec85541410.png)

**Reply by tangming2010** [reply]
yarn workspace @designable/formily-antd start

**Reply by CoderDMBai** [reply]
您好！clone地址后顺序执行了yarn install、yarn build、yarn workspace @formily/designable-antd start，在执行yarn workspace @formily/designable-antd start后报了这个错误，麻烦您看看我该如何解决这个问题呢，谢谢
![image](https://user-images.githubusercontent.com/30764148/151652191-673e8555-4aac-49b8-a764-67d5dcf85f75.png)

**Reply by yaximi111** [reply]
cd 到antd文件夹里面再yarn start也是一样的
